setup.py

```diff
@@ -371,6 +371,7 @@ def find_packages(lib_directory):
         ###############
         'expander': [],
         'filter': [],
+        # 'lcov': [],
```

Comment:
there is another lcov value in this dictonary (extras_require). Should we name it differently?